pkg/render/common/kibana/service.go

```diff
@@ -0,0 +1,20 @@
+package kibana
```

Comment:
I think we can keep elasticsearch and kibana in the same package for now, they're pretty tightly coupled together right now so this could cause circular dependencies.

We should definitely evaluate if this should still be the case once the render refactoring is done.

Reply:
I retract this comment, there is no kibana stuff in the elasticsearch package and this is absolutely the right thing to do.